torchao/dtypes/utils.py

```diff
@@ -37,29 +44,29 @@ def wrapper(*args, **kwargs):
 """
 _LAYOUT_CONSTRUCTOR_TABLE: Dict[Callable, Dict[str, Callable]] = defaultdict(dict)
 
-def _register_layout_cls(cls: Callable, extended_layout: str):
+def _register_layout_cls(cls: Callable, layout_name: str):
```

Comment:
I thought the main benefit of this PR was no longer having to use layout_name or stringly interfaces at all

Reply:
sure I can replace this with the type (class) for LayoutType I think